components/butterfly/src/rumor.rs

```diff
@@ -376,16 +376,16 @@ mod storage {
         {
             let map = self.0.list.read();
             let inner_map = map.get(Departure::const_key());
-            let len = if inner_map.is_some() {
-                inner_map.unwrap().len()
+            let len = if let Some(im) = inner_map {
+                im.len()
```

Comment:
This is a good change!

OPTIONAL: I think we could make this even simpler by replacing L379–383 with:
```rust
            let len = inner_map.map_or(0, HashMap::len);
```

Reply:
Nice, good call